packages/@storefront/flux-capacitor/test/unit/core/reducers/data/navigations.ts

```diff
@@ -199,15 +199,90 @@ suite('navigations', ({ expect }) => {
       expect(reducer).to.eql(newState);
     });
 
-    it('should return state on SELECT_REFINEMENT if no navigationId and refinementIndex', () => {
+    it('should return state on SELECT_REFINEMENT if no navigationId', () => {
       const reducer = navigations(state, <any>{
         type: Actions.SELECT_REFINEMENT,
-        payload: {}
+        payload: { index: 1 },
+      });
+
+      expect(reducer).to.eql(state);
```

Comment:
If we're testing for pass through, this should be `to.eq()`.

Reply:
Done.